commit 525e19f747677991eae7463053b28b264bb986aa (refs/changes/92/4492/2)
Author: Josh Wilsdon <josh@wilsdon.ca>
Date:   2018-07-09T12:47:44-07:00 (1 year, 3 months ago)
    
    TRITON-578 CNS should not continue as though it succeeded in the case of getNapiPools failure
    Reviewed by: Trent Mick <trentm@gmail.com>
    Approved by: Trent Mick <trentm@gmail.com>

diff --git a/CHANGES.md b/CHANGES.md
index 78b1241..fc845c8 100644
--- a/CHANGES.md
+++ b/CHANGES.md
@@ -10,6 +10,10 @@
 
 # triton-cns
 
+## 0.1.2
+
+- TRITON-578 CNS should not continue as though it succeeded in the case of getNapiPools failure
+
 ## 0.1.1
 
 - TRITON-519 fix cns service crash when vm does not have an owner_uuid
diff --git a/lib/api-server.js b/lib/api-server.js
index 1619fb6..bc8f6c0 100644
--- a/lib/api-server.js
+++ b/lib/api-server.js
@@ -3,7 +3,7 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/.
  *
- * Copyright (c) 2015, Joyent, Inc.
+ * Copyright (c) 2018, Joyent, Inc.
  */
 
 module.exports = APIServer;
@@ -612,6 +612,7 @@ function postSuffixesForVM_v1(req, res, next) {
 		if (err) {
 			res.send(500, err);
 			next();
+			return;
 		}
 
 		var vm = {};
diff --git a/package.json b/package.json
index 5137946..9921cb7 100644
--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "triton-cns",
-  "version": "0.1.1",
+  "version": "0.1.2",
   "description": "Trition container naming service",
   "main": "index.js",
   "scripts": {
